src/Testcontainers.EventHubs/EventHubsBuilder.cs

```
@@ -139,7 +139,10 @@ protected override EventHubsBuilder Init()
             .WithPortBinding(EventHubsPort, true)
             .WithPortBinding(EventHubsHttpPort, true)
             .WithWaitStrategy(Wait.ForUnixContainer().UntilHttpRequestIsSucceeded(request =>
-                request.ForPort(EventHubsHttpPort).ForPath("/health")));
+                request.ForPort(EventHubsHttpPort).ForPath("/health"),
+                // If the configuration is invalid, the container will stop immediately.
+                // In this case, we need no to wait for the container to be ready indefinitely.
+                waitStrategyModifier => waitStrategyModifier.WithTimeout(TimeSpan.FromSeconds(30))));
```

Could we please revert this change? As mentioned in the issue, we need to address this differently. We do not set a default timeout on purpose because we do not know anything about the user's environment, hardware, etc.

Instead, we should throw an exception if the container stops during the wait strategy check, but this should be handled in a separate PR.

---

done